dev-python/rtimulib/rtimulib-7.2.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Binding for RTIMULib, a versatile IMU library"
+HOMEPAGE="https://github.com/RPi-Distro/RTIMULib"
+SRC_URI="https://github.com/RPi-Distro/RTIMULib/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~arm ~arm64"
+IUSE="test"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
I'm confused. If it's a Python binding to a library, don't you need to dep on the library?

Reply:
Apparently not. All I can say, when I wan't to use `sense-hat`, which needs `rtimulib`, emerging this one is fine. `sense-hat` then works fine on my RPi3+. So no extra lib required. The description i